### ISSUE TITLE
text formatting infobubble is difficult to read

### ISSUE THREAD

**Issue by Hugo-Trentesaux** [umap-project/umap#813]
When entering a description for a marker, a help bubble displays:

```
Text formatting

    *simple star for italic*
    **double star for bold**
    # one hash for main heading
    ## two hashes for second heading
    ### three hashes for third heading
    Simple link: [[http://example.com]]
    Link with text: [[http://example.com|text of the link]]
    Image: {{http://image.url.com}}
    Image with custom width (in px): {{http://image.url.com|width}}
    Iframe: {{{http://iframe.url.com}}}
    Iframe with custom height (in px): {{{http://iframe.url.com|height}}}
    Iframe with custom height and width (in px): {{{http://iframe.url.com|height*width}}}
    --- for an horizontal rule
```

This is difficult to read for inexperienced users not familiar with this kind of markup. Maybe setting the special characters to bold could help identifying them. It has been reported to me that the vertical bar in the "Link with text" is difficult to see.

**[[** http://example.com **|** text of the link **]]** or even color would help to understand.

**Comment by Spielmops** [comment]
Zoom in with the browser like in any webpage: CTRL and + or CTRL and mousewheel

**Comment by Hugo-Trentesaux** [comment]
Zooming in will not highlight the markdown signs as I suggest. Bold font should be applied instead.

**Comment by Spielmops** [comment]
You are right, but if I zoom in, everything is good to read. On the other hand: a little bit of color enhances the readability - as known from editors with syntax-highlighting ...
Spielmops

**Comment by PeskyPotato** [comment]
I've attempted to make the modal description a bit more verbose so unfamiliar users get a better understanding. I've also added more formatting to the text to improve readability. Not sure if this would help at all but a starting point.

![screenshot of modal each category has a header, description and example](https://github.com/umap-project/umap/assets/35942108/f87ac78b-7df3-4340-9a18-55f92f6b5b2c)

This would mean the translations need to be redone in each language again.

**Comment by PeskyPotato** [comment]
> I'm a bit afraid about the size of the box with such details 

This was my concern as well, on smaller screens it can be a bit much to have all that text. I'll try and condense it a little bit without trying to take out any details to see how it looks, later today. Maybe the solution might just bee to have a similar output to what was there before but a link to the documentation with a more verbose description of each.

**Comment by Spielmops** [comment]
Thank you all, looks much better now.

Spielmops

**Comment by yohanboniface** [comment]
Idea: keep it simple in the modal, and add a link to the discover site, with more details (https://discover.umap-project.org/support/faq/#text-formatting).

Just a downside: the discover site has less languages than uMap code itself.

**Comment by Aurelie-Jallut** [comment]
I like the idea of categorising to help us find what we're looking for

![Image](https://github.com/user-attachments/assets/f93f8402-2481-4ecc-b7fc-6d07a84e67ec)

**Comment by davidbgk** [comment]
It has to fit in a screen, whatever its size 😈 How do we deal with that? Scrolling?

**Comment by Aurelie-Jallut** [comment]
I've tried on my desktop screen, it works without scrolling but this need more test i suppose

![Image](https://github.com/user-attachments/assets/3622874f-93b0-4f1e-b1d3-b85de14d1720)
